slides/intro-02-data-budget.qmd

```diff
@@ -367,5 +367,5 @@ bind_rows(
 ```{r diagram-split, echo = FALSE}
 #| fig-align: "center"
 
-knitr::include_graphics("images/whole-game-split.jpg")
```

Comment:
Since we are using `transparent` versions, do we need to delete the original?

we might be doing it in this PR but I can't tell

Reply:
We are still using the non-transparent version in the build-up in the intro-01-introduction deck so those remain in the repo